### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by ErikStoupignan**
# Add forms 
This branch was used to add forms in the following functions:
- Likes. ✅ 
- Comments. ✅
- Create a new Post. ✅

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -9,6 +9,9 @@
   # get '/users/:user_id/posts/:id', to: 'posts#show', as: :user_post
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :likes, only: [:create]
+      resources :comments, only: [:create]
```
Comment (FrederickMih) [inline config/routes.rb]: - [x] Hi @ErikStoupignan, congrats :tada: you are doing so well in this project. Let's kindly adjust his little issue. There should be a pair of routes for creating comments.
```ruby
  POST   /posts/:post_id/comments(.:format)     comments#create
 GET   /posts/:post_id/comments/new(.:format)     comments#new
```
This requirement is essential for the approval :heavy_check_mark:  of this milestone.

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
```
Comment (FrederickMih) [inline app/controllers/comments_controller.rb:1]: - [x] I think we need the `new` action in this controller. Based on the requirements of this project, there should be a form in the views/comments/new.html.erb file which accepts inputs for creating comments. You did a great job :+1: by creating a partial in this path(views/shared/_add_comment.html.erb) for adding comments. Hence the partial should be rendered in this file.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @ErikStoupignan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


Screenshot from 2022-10-20 09-38-51


## Highlights :green_circle: 

- Posts and comments are created on behave of the currently logged-in user :+1: 
- PR is descriptive :+1: 
- Great commit history :+1: 
- No Linter error :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (ErikStoupignan) [reply config/routes.rb]: Thanks for going through the topic with me on Slack my friend.

I'll add the method "new" in the comments controller as you recommended. (Although it is not a project requirement but to improve the organization)

File: app/controllers/comments_controller.rb (hunk 2 above)
Comment (ErikStoupignan) [reply app/controllers/comments_controller.rb:1]: image

Thanks for taking it up with me on Slack, my friend.

I'll add the "new" method in the comment controller as you recommended. (Although it is not a requirement of the project but to improve the organization)

**Review by julie-ify — APPROVED** [review]
Hi @ErikStoupignan,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**